### ISSUE TITLE
Running sway from tty causes monitor to sleep.

### ISSUE THREAD

**Issue by AustinHasten** [swaywm/sway#379]
When running `sway` from a tty, my monitor immediately goes to sleep. Everything in the output from `sway -d` seems fine except for the first line, `[wlc] failed to open x11 display`. I'm able to start sway from an x session without any noticeable problems, though the output does tell me `libEGL warning: DRI2: failed to authenticate.`

I'm on a fresh install of arch using the nouveau driver. My graphics cards are 2 GTX970s in SLI, connected to my UHD monitor via displayport 1.4.

Here's the output from `# fbset -i`

```
mode "3840x2160"
    geometry 3840 2160 3840 2160 32
    timings 0 0 0 0 0 0 0
    accel true
    rgba 8/16,8/8,8/0,0/0
endmode

Frame buffer device information:
    Name        : nouveaufb
    Address     : 0xe0220000
    Size        : 33177600
    Type        : PACKED PIXELS
    Visual      : TRUECOLOR
    XPanStep    : 1
    YPanStep    : 1
    YWrapStep   : 0
    LineLength  : 15360
    Accelerator : No
```

**Comment by SirCmpwn** [comment]
I'm not sure that multiple graphics cards are well supported on wlc yet (paging @Cloudef).

**Comment by SirCmpwn** [comment]
Try setting WLC_DRM_DEVICE=card1.

**Comment by snarfsnarf222** [comment]
After doing that, sway -d gives me no errors but my sway does not start. I get 
`[wlc] Connector 0 is not connected or has no modes` for connectors 0-4.

**Comment by snarfsnarf222** [comment]
Also, I have to comment out `output * bg /usr/share/sway/wallpaper.jpg fill` from the default config or I get
```
handling config command 'output * bg /usr/share/sway/wallpaper.jpg fill'
Error: Signal 11. Printing backtrace
Backtrace: sway(error_handler+0x2f) [0x41d93f]
Backtrace: /usr/lib/libc.so.6(+0x33680) [0x7f831fc8f680]
Backtrace: sway() [0x40adec]
Backtrace: sway(config_command+0x174) [0x407284]
Backtrace: sway(read_config+0x177) [0x40d517]
Backtrace: sway(load_config+0x12a) [0x40db2a]
Backtrace: sway(main+0x2e3) [0x406513]
Backtrace: /usr/lib/libc.so.6(__libc_start_main+0xf0) [0x7f831fc7c610]
Backtrace: sway(_start+0x29) [0x4065e9]
```

**Comment by Cloudef** [comment]
Yeah I'm not really sure if SLI setup works at all (as I don't have one), WLC_DRM_DEVICE may help.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
